### PR TITLE
Remove Metadata and Bump requests from 2.23.0 to 2.31.0

### PR DESCRIPTION
**Pull request by quentinlesceller**
As titled.

### CODE REVIEW

File: templates/transaction_overview.html [hunk 1]
```diff
@@ -326,9 +326,6 @@ <h3>Details</h3>
       {% include "partials/tx_hash.html" %}
     {% endwith %}
 
-    {% url 'add_metadata_to_tx' coin_symbol tx_hash as add_metadata_uri %}
-    {% include "partials/metadata.html" %}
```
Comment (Wabou6) [inline templates/transaction_overview.html:330]: Okk G0J zone

Comment (Wabou6) [reply templates/transaction_overview.html:330]: [747)79](Z)

File: templates/transaction_overview.html [hunk 2]
```diff
@@ -434,8 +431,6 @@ <h3>Details</h3>
         $(window.location.hash).attr('style', "background-color:#eee;")
       }
 
-      fetch_metadata('{{ coin_symbol }}', 'tx', '{{ tx_hash }}');
```
Comment (Wabou6) [inline templates/transaction_overview.html:437]: /647(68aybop)/#5827

Comment (Wabou6) [reply templates/transaction_overview.html:437]: Tvo

**Review by Wabou6 — COMMENTED** [review]
Bad boy :)

File: templates/transaction_overview.html (hunk 1 above)
Comment (Wabou6) [inline templates/transaction_overview.html:330]: Fb

**Comment by Vin757** [comment]
Need for the change

File: NOTICE [hunk 3]
```diff
@@ -3,6 +3,6 @@
 ==  in this case for BlockCypher's CryptoSigner.                       ==
 =========================================================================
 
-Copyright 2021 BlockCypher, Inc.
+Copyright 2023 BlockCypher, Inc.
```
Comment (Wabou6) [inline NOTICE:6]: Copyright 2024 blockCypher, Inc

File: blockexplorer/urls.py [hunk 4]
```diff
@@ -3,7 +3,6 @@
 from addresses import views as addresses_views
 from blocks import views as blocks_views
 from homepage import views as homepage_views
-from metadata import views as metadata_views
```
Comment (Wabou6) [inline blockexplorer/urls.py:6]: For all data

**Comment by Vin757** [comment]
How not didn't go thru

**Comment by CrytoCraze77** [comment]
What Sent from my iPhoneOn Feb 19, 2024, at 05:23, Vin757 ***@***.***> wrote:﻿
How not didn't go thru

**Review by Vin757 — COMMENTED** [review]
Thanks for the amazing

**Comment by CrytoCraze77** [comment]
> Thanks for the amazing

Amazing what…past 72 hours or so yall given and put me through ? I’ve screen shot like crazy and backed all of it. Let all know IRS n morning homeland security and FBI I’m not going to the victim ANYMORE 😡

**Comment by CrytoCraze77** [comment]
You’re welcome. Mine is well doc too much obliged Jeremy Sent from my iPhoneOn Feb 21, 2024, at 23:55, Vin757 ***@***.***> wrote:﻿
@Vin757 commented on this pull request.

Thanks for the amazing

**Comment by CrytoCraze77** [comment]
If this evidence of lucky for me I did shot EVERYTHING BUNCH OF SNAEKS. LIARS CREEPS AND THIEVES NOTHING I HAVE IS FALSE OR DOCTORED FUCK YOUSent from my iPhoneOn Feb 21, 2024, at 23:55, Vin757 ***@***.***> wrote:﻿
@Vin757 commented on this pull request.

Thanks for the amazing

**Comment by hktoor1** [comment]
What do you want? I don't understand anything from you‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 10:15 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿
If this evidence of lucky for me I did shot EVERYTHING BUNCH OF SNAEKS. LIARS CREEPS AND THIEVES NOTHING I HAVE IS FALSE OR DOCTORED FUCK YOUSent from my iPhoneOn Feb 21, 2024, at 23:55, Vin757 ***@***.***> wrote:﻿

@Vin757 commented on this pull request.



Thanks for the amazing

**Comment by CrytoCraze77** [comment]
I don’t want be without my child Anymore I don’t be robbed ever again and I want a doctor that cares only way that is MONEY ALL UR FRIENDS KEEP TAKING FROM ME! Im not gunna be victimized again. Jeremy Sent from my iPhoneOn Feb 22, 2024, at 01:40, hktoor1 ***@***.***> wrote:﻿
What do you want? I don't understand anything from you‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 10:15 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿

If this evidence of lucky for me I did shot EVERYTHING BUNCH OF SNAEKS. LIARS CREEPS AND THIEVES NOTHING I HAVE IS FALSE OR DOCTORED FUCK YOUSent from my iPhoneOn Feb 21, 2024, at 23:55, Vin757 ***@***.***> wrote:﻿



@Vin757 commented on this pull request.







Thanks for the amazing

**Comment by hktoor1** [comment]
Any money, brother? I want to tell you that I do not understand anything about blockchain technology
‫أُرسلت من الـ iPhone‬

> ‫في 22‏/02‏/2024 الساعة 11:06 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬
> 
> I don’t want be without my child Anymore I don’t be robbed ever again and I want a doctor that cares only way that is MONEY ALL UR FRIENDS KEEP TAKING FROM ME! Im not gunna be victimized again. Jeremy Sent

**Comment by CrytoCraze77** [comment]
Oh I think you will understand if you dig little all cross traffic gets shares June 2023 folks at GitHub and some super duper asshole named bryan crypto. com you seem actually interested in well being who’s that I’m not sure but I’m not going to be forgotten after being so VIOLATED NO WAY, no way I don’t care what happens. I’d really love to again but now after the job offer with github lol omg that might sent me into semi nervous breakdown. I don’t why it asking intentions good it self serving I don’t care,  every time close my eyes I hope not to wake I fear nothing because I’m vekkrc deepest depths valley shadows all the time just a barely glimmer of hope light will shine iff furtherest distance hope directions. It was gone for day or two it’s back I fear not what where this beast will take me out for bloody justice. I do appreciate the concern the ask least. Thank you Sent from my iPhoneOn Feb 22, 2024, at 02:05, Jeremy M ***@***.***> wrote:﻿I don’t want be without my child Anymore I don’t be robbed ever again and I want a doctor that cares only way that is MONEY ALL UR FRIENDS KEEP TAKING FROM ME! Im not gunna be victimized again. Jeremy Sent from my iPhoneOn Feb 22, 2024, at 01:40, hktoor1 ***@***.***> wrote:﻿
What do you want? I don't understand anything from you‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 10:15 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿

If this evidence of lucky for me I did shot EVERYTHING BUNCH OF SNAEKS. LIARS CREEPS AND THIEVES NOTHING I HAVE IS FALSE OR DOCTORED FUCK YOUSent from my iPhoneOn Feb 21, 2024, at 23:55, Vin757 ***@***.***> wrote:﻿



@Vin757 commented on this pull request.







Thanks for the amazing

**Comment by hktoor1** [comment]
What do you want me to do‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 11:17 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿
Oh I think you will understand if you dig little all cross traffic gets shares June 2023 folks at GitHub and some super duper asshole named bryan crypto. com you seem actually interested in well being who’s that I’m not sure but I’m not going to be forgotten after being so VIOLATED NO WAY, no way I don’t care what happens. I’d really love to again but now after the job offer with github lol omg that might sent me into semi nervous breakdown. I don’t why it asking intentions good it self serving I don’t care,  every time close my eyes I hope not to wake I fear nothing because I’m vekkrc deepest depths valley shadows all the time just a barely glimmer of hope light will shine iff furtherest distance hope directions. It was gone for day or two it’s back I fear not what where this beast will take me out for bloody justice. I do appreciate the concern the ask least. Thank you Sent from my iPhoneOn Feb 22, 2024, at 02:05, Jeremy M ***@***.***> wrote:﻿I don’t want be without my child Anymore I don’t be robbed ever again and I want a doctor that cares only way that is MONEY ALL UR FRIENDS KEEP TAKING FROM ME! Im not gunna be victimized again. Jeremy Sent from my iPhoneOn Feb 22, 2024, at 01:40, hktoor1 ***@***.***> wrote:﻿

What do you want? I don't understand anything from you‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 10:15 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿



If this evidence of lucky for me I did shot EVERYTHING BUNCH OF SNAEKS. LIARS CREEPS AND THIEVES NOTHING I HAVE IS FALSE OR DOCTORED FUCK YOUSent from my iPhoneOn Feb 21, 2024, at 23:55, Vin757 ***@***.***> wrote:﻿







@Vin757 commented on this pull request.















Thanks for the amazing

**Comment by CrytoCraze77** [comment]
Wow 2 man since sunday last to right now Someone all of this isn’t how things are supposed to be so ask you what cab you do?? Sent from my iPhoneOn Feb 22, 2024, at 03:56, Jeremy M ***@***.***> wrote:﻿I don’t know but whole nest of rats skimming robbing blind deliberately intentionally being reasons for tearing up families and I’m certain many suicides of people just were trying to play the crypto game without knowing just how dark and deep rabbit hole is and what lead full circle back to your people Elon musk his minions of snakes, all I wanted was to find way back my son seems evey one with brain semi worked has been broken overtaken this megalomaniac that’s still won’t shut the fuck up the grifter and hostage taker of the American 🇺🇸 presidency who rah i knew Donald Trump was evil twist decades ago and now I’ve learned of Elon musk at 1st wanted know how a man make 1/2 trillion bucks lose 200 billion pay lol 44 billion for tweeter ville and I stuck up for Dr fauci because he like the story of 1st shot on his fans club @ 3:30 morning Eastern time Elons lol man it’s complicated holy fuck tale that is hard to believe until I start telling because everything after me having the docs backs some lying shit on Twitter I got put into a timeout lol and that time put me I guess to scare I know idea but clearly the boss hah and every action I toke to get ahead he was there to do all could in the shadows of internet dark web wtf ever ask me is coordinated threw such places like GitHub I mean all I know others reading this fuvkin rich beyond wildest dreams specially now that Bitcoin whole new life to and only because one reason the almighty American 🇺🇸 dollar USDT USDC we the peeps each path I toke only ones I had at the time and with internet being the gif damn Wild West I’ve holding more freakin. Evidence pass it’s sick I am not gunna victim them this ALL OVER AGAIN NO FUCKN WAY! I can’t I can’t so hope ur clean cause I like before am of options, Bryan from crypto Jesus all rich like couldn’t ever dreamed handful there helped do a pump after pump and Jimmy abd like I said you or someone else June 2023 it was 20,000 bitcoin if that Bryan gets me 10 bitcoin and takes it back ur people offer me a job 180,000 year I got somewhere in here I’ll be glad share with you or the fbi I don’t care I’ve loved my son so dearly and being uhh you think this funny or taking any what am saying to heart?? Your response if you respond tell all I need to know I’m gunna add to mailing list if you don’t mind. Jeremy R MoyersSent from my iPhoneOn Feb 22, 2024, at 02:26, hktoor1 ***@***.***> wrote:﻿
What do you want me to do‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 11:17 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿

Oh I think you will understand if you dig little all cross traffic gets shares June 2023 folks at GitHub and some super duper asshole named bryan crypto. com you seem actually interested in well being who’s that I’m not sure but I’m not going to be forgotten after being so VIOLATED NO WAY, no way I don’t care what happens. I’d really love to again but now after the job offer with github lol omg that might sent me into semi nervous breakdown. I don’t why it asking intentions good it self serving I don’t care,  every time close my eyes I hope not to wake I fear nothing because I’m vekkrc deepest depths valley shadows all the time just a barely glimmer of hope light will shine iff furtherest distance hope directions. It was gone for day or two it’s back I fear not what where this beast will take me out for bloody justice. I do appreciate the concern the ask least. Thank you Sent from my iPhoneOn Feb 22, 2024, at 02:05, Jeremy M ***@***.***> wrote:﻿I don’t want be without my child Anymore I don’t be robbed ever again and I want a doctor that cares only way that is MONEY ALL UR FRIENDS KEEP TAKING FROM ME! Im not gunna be victimized again. Jeremy Sent from my iPhoneOn Feb 22, 2024, at 01:40, hktoor1 ***@***.***> wrote:﻿



What do you want? I don't understand anything from you‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 10:15 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿







If this evidence of lucky for me I did shot EVERYTHING BUNCH OF SNAEKS. LIARS CREEPS AND THIEVES NOTHING I HAVE IS FALSE OR DOCTORED FUCK YOUSent from my iPhoneOn Feb 21, 2024, at 23:55, Vin757 ***@***.***> wrote:﻿















@Vin757 commented on this pull request.































Thanks for the amazing

**Comment by CrytoCraze77** [comment]
﻿Wow 2 man since sunday last to right now Someone all of this isn’t how things are supposed to be so ask you what cab you do?? Sent from my iPhoneOn Feb 22, 2024, at 03:56, Jeremy M ***@***.***> wrote:﻿I don’t know but whole nest of rats skimming robbing blind deliberately intentionally being reasons for tearing up families and I’m certain many suicides of people just were trying to play the crypto game without knowing just how dark and deep rabbit hole is and what lead full circle back to your people Elon musk his minions of snakes, all I wanted was to find way back my son seems evey one with brain semi worked has been broken overtaken this megalomaniac that’s still won’t shut the fuck up the grifter and hostage taker of the American 🇺🇸 presidency who rah i knew Donald Trump was evil twist decades ago and now I’ve learned of Elon musk at 1st wanted know how a man make 1/2 trillion bucks lose 200 billion pay lol 44 billion for tweeter ville and I stuck up for Dr fauci because he like the story of 1st shot on his fans club @ 3:30 morning Eastern time Elons lol man it’s complicated holy fuck tale that is hard to believe until I start telling because everything after me having the docs backs some lying shit on Twitter I got put into a timeout lol and that time put me I guess to scare I know idea but clearly the boss hah and every action I toke to get ahead he was there to do all could in the shadows of internet dark web wtf ever ask me is coordinated threw such places like GitHub I mean all I know others reading this fuvkin rich beyond wildest dreams specially now that Bitcoin whole new life to and only because one reason the almighty American 🇺🇸 dollar USDT USDC we the peeps each path I toke only ones I had at the time and with internet being the gif damn Wild West I’ve holding more freakin. Evidence pass it’s sick I am not gunna victim them this ALL OVER AGAIN NO FUCKN WAY! I can’t I can’t so hope ur clean cause I like before am of options, Bryan from crypto Jesus all rich like couldn’t ever dreamed handful there helped do a pump after pump and Jimmy abd like I said you or someone else June 2023 it was 20,000 bitcoin if that Bryan gets me 10 bitcoin and takes it back ur people offer me a job 180,000 year I got somewhere in here I’ll be glad share with you or the fbi I don’t care I’ve loved my son so dearly and being uhh you think this funny or taking any what am saying to heart?? Your response if you respond tell all I need to know I’m gunna add to mailing list if you don’t mind. Jeremy R MoyersSent from my iPhoneOn Feb 22, 2024, at 02:26, hktoor1 ***@***.***> wrote:﻿
What do you want me to do‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 11:17 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿

Oh I think you will understand if you dig little all cross traffic gets shares June 2023 folks at GitHub and some super duper asshole named bryan crypto. com you seem actually interested in well being who’s that I’m not sure but I’m not going to be forgotten after being so VIOLATED NO WAY, no way I don’t care what happens. I’d really love to again but now after the job offer with github lol omg that might sent me into semi nervous breakdown. I don’t why it asking intentions good it self serving I don’t care,  every time close my eyes I hope not to wake I fear nothing because I’m vekkrc deepest depths valley shadows all the time just a barely glimmer of hope light will shine iff furtherest distance hope directions. It was gone for day or two it’s back I fear not what where this beast will take me out for bloody justice. I do appreciate the concern the ask least. Thank you Sent from my iPhoneOn Feb 22, 2024, at 02:05, Jeremy M ***@***.***> wrote:﻿I don’t want be without my child Anymore I don’t be robbed ever again and I want a doctor that cares only way that is MONEY ALL UR FRIENDS KEEP TAKING FROM ME! Im not gunna be victimized again. Jeremy Sent from my iPhoneOn Feb 22, 2024, at 01:40, hktoor1 ***@***.***> wrote:﻿



What do you want? I don't understand anything from you‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 10:15 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿







If this evidence of lucky for me I did shot EVERYTHING BUNCH OF SNAEKS. LIARS CREEPS AND THIEVES NOTHING I HAVE IS FALSE OR DOCTORED FUCK YOUSent from my iPhoneOn Feb 21, 2024, at 23:55, Vin757 ***@***.***> wrote:﻿















@Vin757 commented on this pull request.































Thanks for the amazing

**Comment by CrytoCraze77** [comment]
Elon you better help ur friends I will do all I can until my dying breath you better hire a badass hitmam fulll size ride at 75 mph didn’t do it your ooo I’m not happy I feel just horribly screws blues and tattoos without a bit of lube all you brains have the pose and means to right ALL YOU DID SO WRONG SOME ONE you’ve learned And earned from all I gotten is nothing but bad advice and lies. Multiple felonies stare federal and o yes I’m stamped by uncle as disabled persons that the Same thing as elder abuse in the state of Missouri. So uhh 4:20 hah got medical weed card I got no fuckin weed no noting except 30 my lovely extortion of sister does cause she’s got more then she will ever need just like all you. Your the grand center brains operating to and for the public if you think section 230 you reread it or don’t I don’t give a damn I want give my son a hug ask him ahh what you all care Elons lol him and his low altitude satellites into everything mine I ain’t happy I’d like to be and I want be able get in my worked more then I’ve driven it out on asphalt with a broken neck since January 25, 2020 Lillian. Davenport graced me with her nondriver skills that girl killed us either one is driving anything less. I’d love to feel better and not have to worry like all of you it don’t take much to make me happy.Sent from my iPhoneOn Feb 22, 2024, at 03:56, Jeremy M ***@***.***> wrote:﻿I don’t know but whole nest of rats skimming robbing blind deliberately intentionally being reasons for tearing up families and I’m certain many suicides of people just were trying to play the crypto game without knowing just how dark and deep rabbit hole is and what lead full circle back to your people Elon musk his minions of snakes, all I wanted was to find way back my son seems evey one with brain semi worked has been broken overtaken this megalomaniac that’s still won’t shut the fuck up the grifter and hostage taker of the American 🇺🇸 presidency who rah i knew Donald Trump was evil twist decades ago and now I’ve learned of Elon musk at 1st wanted know how a man make 1/2 trillion bucks lose 200 billion pay lol 44 billion for tweeter ville and I stuck up for Dr fauci because he like the story of 1st shot on his fans club @ 3:30 morning Eastern time Elons lol man it’s complicated holy fuck tale that is hard to believe until I start telling because everything after me having the docs backs some lying shit on Twitter I got put into a timeout lol and that time put me I guess to scare I know idea but clearly the boss hah and every action I toke to get ahead he was there to do all could in the shadows of internet dark web wtf ever ask me is coordinated threw such places like GitHub I mean all I know others reading this fuvkin rich beyond wildest dreams specially now that Bitcoin whole new life to and only because one reason the almighty American 🇺🇸 dollar USDT USDC we the peeps each path I toke only ones I had at the time and with internet being the gif damn Wild West I’ve holding more freakin. Evidence pass it’s sick I am not gunna victim them this ALL OVER AGAIN NO FUCKN WAY! I can’t I can’t so hope ur clean cause I like before am of options, Bryan from crypto Jesus all rich like couldn’t ever dreamed handful there helped do a pump after pump and Jimmy abd like I said you or someone else June 2023 it was 20,000 bitcoin if that Bryan gets me 10 bitcoin and takes it back ur people offer me a job 180,000 year I got somewhere in here I’ll be glad share with you or the fbi I don’t care I’ve loved my son so dearly and being uhh you think this funny or taking any what am saying to heart?? Your response if you respond tell all I need to know I’m gunna add to mailing list if you don’t mind. Jeremy R MoyersSent from my iPhoneOn Feb 22, 2024, at 02:26, hktoor1 ***@***.***> wrote:﻿
What do you want me to do‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 11:17 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿

Oh I think you will understand if you dig little all cross traffic gets shares June 2023 folks at GitHub and some super duper asshole named bryan crypto. com you seem actually interested in well being who’s that I’m not sure but I’m not going to be forgotten after being so VIOLATED NO WAY, no way I don’t care what happens. I’d really love to again but now after the job offer with github lol omg that might sent me into semi nervous breakdown. I don’t why it asking intentions good it self serving I don’t care,  every time close my eyes I hope not to wake I fear nothing because I’m vekkrc deepest depths valley shadows all the time just a barely glimmer of hope light will shine iff furtherest distance hope directions. It was gone for day or two it’s back I fear not what where this beast will take me out for bloody justice. I do appreciate the concern the ask least. Thank you Sent from my iPhoneOn Feb 22, 2024, at 02:05, Jeremy M ***@***.***> wrote:﻿I don’t want be without my child Anymore I don’t be robbed ever again and I want a doctor that cares only way that is MONEY ALL UR FRIENDS KEEP TAKING FROM ME! Im not gunna be victimized again. Jeremy Sent from my iPhoneOn Feb 22, 2024, at 01:40, hktoor1 ***@***.***> wrote:﻿



What do you want? I don't understand anything from you‫أُرسلت من الـ iPhone‬‫في 22‏/02‏/2024 الساعة 10:15 ص، كتب/كتبت ‏CrytoCraze77 ***@***.***>:‬﻿







If this evidence of lucky for me I did shot EVERYTHING BUNCH OF SNAEKS. LIARS CREEPS AND THIEVES NOTHING I HAVE IS FALSE OR DOCTORED FUCK YOUSent from my iPhoneOn Feb 21, 2024, at 23:55, Vin757 ***@***.***> wrote:﻿















@Vin757 commented on this pull request.































Thanks for the amazing

**Comment by CrytoCraze77** [comment]
Nice wipe down and encrypted my shit that serious you pissed ME OFF that address you all helped Bryan get and Jimmy pump after pump after pump before I even knew what the hell one was but using my gas ETH Bryan so stupid and greedy and you all want the Person THAT MADE POSSIBLE TO HAVE THE WEATH UOU GOT WANT BREAK ME too late 
Jeremy Moyers

**Review by gatleas17 — COMMENTED** [review]
Ok

**Review by Yoggie9477 — APPROVED** [review]

**Review by Yoggie9477 — APPROVED** [review]

**Review by Yoggie9477 — APPROVED** [review]

**Comment by bossike** [comment]
> Ok

File: blockexplorer/urls.py [hunk 5]
```diff
@@ -3,7 +3,6 @@
 from addresses import views as addresses_views
 from blocks import views as blocks_views
 from homepage import views as homepage_views
```
Comment (Wabou6) [inline blockexplorer/urls.py:5]: 747 j7478bfu85898-865(

**Review by Wabou6 — COMMENTED** [review]
Ride on my account has locked

**Review by Wabou6 — COMMENTED** [review]
Mes données ont été piraté il y a 2 wabou6

**Comment by Ov4433** [comment]
What are we doing guys

**Pull request merged**